typescript/cli/src/tests/commands/warp.ts

```diff
@@ -45,13 +47,31 @@ export async function hyperlaneWarpRead(
 ) {
   return $`yarn workspace @hyperlane-xyz/cli run hyperlane warp read \
         --registry ${REGISTRY_PATH} \
+        --overrides " " \
```

Comment:
Needed for local testing to ignore local registry

Reply:
Makes sense, can you leave a comment above the first one